extra_data/write_cxi.py

```diff
@@ -147,34 +221,40 @@ def collect_data(self):
                 'data': VLayout(shape, dtype=image_grp['data'].dtype),
             }
 
-        layouts['cellId'] = VLayout((self.nframes, self.nmodules),
-                                    dtype=image_grp['cellId'].dtype)
+        layouts[self.cell_id_label] = VLayout(
+            (self.nframes, self.nmodules),
+            dtype=image_grp[self.cell_id_label].dtype
+        )
 
         for name, layout in layouts.items():
-            key = 'image.{}'.format(name)
+            key = '{}.{}'.format(self.group_label, name)
             have_data = np.zeros((self.nframes, self.nmodules), dtype=bool)
 
             for source, modno in self.detdata.source_to_modno.items():
+                print(f" ### Source: {source}, ModNo: {modno}, Key: {key}")
+                module_id = self.modulenos.index(modno)
                 for chunk in self.data._find_data_chunks(source, key):
                     vsrc = h5py.VirtualSource(chunk.dataset)
-                    self._map_chunk(chunk, vsrc, layout, modno, have_data)
+                    self._map_chunk(chunk, vsrc, layout, module_id, have_data)
 
             filled_pct = 100 * have_data.sum() / have_data.size
-            log.info("Assembled %d chunks for %s, filling %.2f%% of the hyperslab",
-                     len(layout.sources), key, filled_pct)
+            log.info(f"Assembled {len(layout.sources):d} chunks for {key:s}, "
+                     f"filling {filled_pct:.2f}% of the hyperslab")
 
         return layouts
 
     def write(self, filename, fillvalues=None):
-        """Write the file on disc to filename
+        """
+        Write the file on disc to filename.
```

Comment:
You don't need to revert these changes now, but for future reference: please try to do formatting changes in a separate PR from changes that affect what the code is doing. It's less work to review the two kinds of change separately rather than in one big diff.

Reply:
Ok, or maybe an additional commit in the same PR after the main PR review finished?

Reply:
Yup, that should be workable as well.